docs/decisions/0017-reimplement-asset-processing.rst

```diff
@@ -23,8 +25,8 @@ The status will be moved to *Accepted* upon completion of reimplementation. Rela
 Context
 *******
 
-State of edx-platform frontends
-===============================
+State of edx-platform frontends (early 2023)
+============================================
```

Comment:
Timestamping this since the line between XModule and XBlock has blurred since writing. Still, the table still makes sense from the perspective of this ADR.